native-wheels-frontend/components/settings/ResetPassword.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { AxiosError } from 'axios';
+import { ThemedView } from '@/components/ThemedView';
+import { ThemedText } from '@/components/ThemedText';
+import StyledInput from '@/components/StyledInput';
+import StyledButton from '@/components/StyledButton';
+import { showErrorToast, showSuccessToast } from '@/components/toast';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+import { useAuth } from '@/contexts/AuthContext';
+import instance from '@/axios/instance'; 
+import { showInfoToast } from "@/components/toast";
+
+export default function ResetPasswordScreen() {
+    const { logout } = useAuth();
+    const [oldPassword, setOldPassword] = useState('');
+    const [newPassword, setNewPassword] = useState('');
+    const [confirmNewPassword, setConfirmNewPassword] = useState('');
+
+    const handleLogout = async () => {
+        logout();
+        showInfoToast("Logged out", "You have been successfully logged out.");
+      };
+
+    const validateInputs = () => {
+        if (!oldPassword || !newPassword || !confirmNewPassword) {
+        showErrorToast('Validation Error', 'Please fill in all fields.');
+        return false;
+        }
+        if (newPassword !== confirmNewPassword) {
+        showErrorToast('Validation Error', 'New passwords do not match.');
+        return false;
+        }
+        return true;
+    };
+
+    const handlePasswordReset = async () => {
+        if (!validateInputs()) return;
+
+        try {
+        await instance.post('/auth/reset-password', { oldPassword, newPassword });
+        showSuccessToast('Success', 'Password has been reset successfully.');
+        setOldPassword('');
+        setNewPassword('');
+        setConfirmNewPassword('');
+        } catch (error) {
+        const axiosError = error as AxiosError;
+        showErrorToast('Error', 'Failed to reset password. Please try again.');
+        console.error(axiosError);
+        }
```

Comment:
Do you have prettier installed? Some of the code looks unformatted with indents etc.